rspotify-model/src/show.rs

```diff
@@ -76,7 +77,6 @@ pub struct FullShow {
     pub publisher: String,
     #[serde(rename = "type")]
     pub _type: String,
```

Comment:
Same here

Reply:
Done. I've tried the endpoint and its value is `"show"`, so it can indeed be removed: https://developer.spotify.com/console/get-show/?id=38bS44xjbVVZ3No3ByF1dJ